web_widget_x2many_2d_matrix/static/src/js/abstract_view_matrix_limit_extend.js

```diff
@@ -0,0 +1,24 @@
+odoo.define('web_widget_x2many_2d_matrix.matrix_limit_extend', function (require) {
+    "use strict";
+
+var AbstractView = require('web.AbstractView');
+
+var AbstractViewCustom = AbstractView.include({
+
+    _setSubViewLimit: function (attrs) {
+	this._super(attrs);
+
+    var view = attrs.views && attrs.views[attrs.mode];
```

Comment:
bad indent :)
Also, for our future selves, please add a comment explaining why we do this ;)